compiler/test-resources/repl/errmsgs

```diff
@@ -72,3 +72,11 @@ scala> val a: iDontExist = 1
 1 | val a: iDontExist = 1
   |        ^^^^^^^^^^
   |        not found: type iDontExist
+scala> def foo1(x: => Int) = x _
```

Comment:
I would make it an error message test instead. They are less fragile

Reply:
But I really want to know what the exact output is here :). Can you help me figure out why the test doesn't pass?